### ISSUE TITLE
CallMolecularConsensusReads generate unmapped BAM

### ISSUE THREAD

**Issue by chadisaad** [fulcrumgenomics/fgbio#530]
when doing CallMolecularConsensusReads , the generated BAM is unmapped ! when doing samtools flagstat, I got 0% mapped read. I also verified the BAM file content.

Do I have to align again with BWA ?

**Comment by nh13** [comment]
Yes, it is very tricky to keep track of and decide on a consensus _mapping_ in addition to consensus _making_, so we decided to make the output reads come out as unmapped.  Typically, the # of consensus reads is **much** smaller than the # of input raw reads, so mapping the consensus reads should not be such a burden.

**Comment by fleharty** [comment]
@nh13 What's a typical fraction of loss that you see.  Sometimes I see 20x loss.

**Comment by nh13** [comment]
@fleharty what do you mean by loss. Do you mean how many consensus reads that don’t map? Number of consensus reads over the number of raw reads?

**Comment by fleharty** [comment]
@nh13 
<img width="707" alt="Screenshot 2019-09-17 09 38 13" src="https://user-images.githubusercontent.com/10714381/65046569-e0844a80-d92e-11e9-86e4-92b9ca7a9e1b.png">

I've attached a plot from a set of reads with family size 11.  In this plot you can see the ba size plotted with respect to the number of observed reads.  The red line is a fit of a beta-binomial model.  The fit is pretty good down to the singletons where we see a much larger number of ba_size = 0 than we expect from the model.

This is one source of loss that we don't really understand.  But back to my original question, I think I should rephrase it to, what is a typical duplex recovery?

I realize that may be an annoying question, because it may vary quite a bit experiment to experiment.

**Comment by nh13** [comment]
@fleharty I don't think we can comment publicly about what we see in our client data, but if you shoot us an email, we can introduce you to folks who can.
